ferrocene/doc/internal-procedures/src/manual-upstream-pulls.rst

```diff
@@ -11,96 +11,60 @@ An "upstream pull" is a pull request that pulls in commits from the upstream Rus
 repository back into the Ferrocene repository to keep them in sync. This
 procedure is usually performed by a periodic Github Action that opens a pull
 request automatically, only requiring sign off of a reviewer to get merged.
-Sometimes merge conflicts may occur in which case automation opens an issue
-describing the conflict and requiring the upstream pull to be done manually.
+Sometimes merge conflicts may occur in which case the conflict needs to to be
+resolved manually.
 
-Manual intervention when the automation fails
```

Comment:
Should I rename the file from `manual-upstream-pulls.rst` to `upstream-pulls.rst`, since it is not manual anymore?

Reply:
Yes!